pages/index.tsx

```diff
@@ -1,16 +1,53 @@
 import Head from 'next/head'
+import Header from 'components/Header'
+import Title from 'components/index/Title'
+import { Container, Typography } from '@mui/material'
 
-export default function Home() {
-  return (
-    <div>
-      <Head>
-        <title>ZIGEN - XR Windowing System</title>
-        <meta name="description" content="ZIGEN: A windowing system for XR" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-      <div className="container mx-auto px-4 mt-4">
-        <h1 className="text-5xl">ZIGEN: Windowing System for XR</h1>
-      </div>
-    </div>
-  )
-}
+const Home = () => (
+  <>
+    <Head>
+      <title>ZIGEN - XR Windowing System</title>
+      <meta name="description" content="ZIGEN - A windowing system for XR" />
+      <link rel="icon" href="/favicon.ico" />
+    </Head>
+
+    <Header />
+    <Title />
+
+    <Container maxWidth="md" className="my-16">
+      <Typography className="indent-2">
+        In the modern desktop environment, windowing systems such as X11 or
+        Wayland provide a multitasking environment where multi applications from
+        different developers work together.
+      </Typography>
+      <Typography className="mt-1 indent-2">
+        However, in today’s immersive environment, a single main application
+        occupies the user’s entire field of view. This limitation was not fatal
+        as long as the primary use of XR was for games that create a unified
+        world.
+      </Typography>
+      <Typography className="mt-1 indent-2">
+        But now, the use of XR is becoming more diverse, and this limitation can
+        no longer be ignored, especially when XR is used as a work environment.
+      </Typography>
+      <Typography className="mt-1 indent-2">
+        We need an XR work environment where users can
+      </Typography>
+      <Typography className="mt-1">
+        ・install and use multiple applications from different developers
```

Comment:
In my opinion, symbols like dots should be specified in CSS styles (+ `<li></li>`), not expressed in plain text, because of the uniformity.
The same is true for line 43.